README.md

```diff
@@ -0,0 +1,195 @@
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [oop-ruby] <a name="about-project"></a>
+
+Object Oriented Programming with RUBY
+
+## 🛠 Built With <a name="built-with"></a>
+
+Ruby
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Ruby</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+Learning how to work with objects on ruby
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Ruby
+
+### Prerequisites
+
+In order to run this project you need:
+
+In order to run this project you need:
+
+✅ Install Ruby <br>
+✅ Create a repo on your repository files <br>
+✅ Clone or make a copy of this repo on your local machine <br>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd oop-ruby
+  git clone https://github.com/gersonhiggins/oop-ruby.git
+```
+
+### Install
+
+Install this project with:
+
+```bash
+bundle install
+```
+
+### Usage
+
+You can use This as ruby oop example
+
+### Run tests
+
+To run tests, run the following command:
+
+```bash
+ rspec spec/*
+```
+
+### Deployment
+
+N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Gerson Higgins**
+
+- GitHub: [@gersonhiggins](https://github.com/gersonhiggins)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+Add console interactivity
+Enable user to create items
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] The issues link is broken kindly consider fixing it so the README file doesn't have any broken links